common/config.go

```diff
@@ -0,0 +1,37 @@
+package common
+
+// Read configuration stuff from a yaml file.
+
+import (
+	"log"
+	"io/ioutil"
+	"gopkg.in/yaml.v2"
+)
+
+type BackendConfig struct {
+	Address string
+	Ip []byte
+	HealthCheck string
```

Comment:
If I understand correctly, `Address` is only meaningful if `HealthCheck` is `"http"`.  We should probably figure out a better type structure here.

Reply:
I guess we could have a better internal representation, but since the field names are all getting directly magically deserialized, I'm not so sure what a better structure for this would look like.